WORKSPACE

```diff
@@ -387,7 +387,6 @@ crates_repository(
         "//:src/persist-cli/Cargo.toml",
         "//:src/persist-client/Cargo.toml",
         "//:src/persist-proc/Cargo.toml",
-        "//:src/persist-txn/Cargo.toml",
```

Comment:
probably needs to be `txn-wal`?

Reply:
Yeah. And I need to get some hash value updated; working on it.